base/fill_struct.go

```diff
@@ -23,9 +23,9 @@ type ArbitrarySetter interface {
 	SetArbitrary(string, interface{}) error
 }
 
-// timeObj and ifaceObj are used for reflect.TypeOf
+// timeObj and strObj are used for reflect.TypeOf
 var timeObj time.Time
-var ifaceObj interface{}
+var strObj string
```

Comment:
I  prefer multiple var declarations grouped by parenthesis. pedantic, but just to keep it consistent:
```go
var (
  // timeObj and strObj are use for reflect.TypeOf
  timeObj time.Time
  strObj string
)
```